modules/cnf-image-based-upgrade-troubleshooting.adoc

```diff
@@ -0,0 +1,289 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-image-based-upgrade-base.adoc
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-troubleshooting_{context}"]
+= Troubleshooting image-based upgrades with {lcao}
+
+You can encounter issues during the image-based upgrade.
+
+[id="ztp-image-based-upgrade-troubleshooting-must-gather_{context}"]
+== Collecting logs
+
+You can use the `oc adm must-gather` CLI to collect information for debugging and troubleshooting.
+
+.Procedure
+
+. Collect data about the Operators by running the following command:
++
+[source,terminal]
+----
+$  oc adm must-gather \
+--dest-dir=must-gather/tmp \
+--image=$(oc -n openshift-lifecycle-agent get deployment.apps/lifecycle-agent-controller-manager -o jsonpath='{.spec.template.spec.containers[?(@.name == "manager")].image}') \
+--image=quay.io/konveyor/oadp-must-gather:latest \ <1>
+--image=quay.io/openshift/origin-must-gather:latest <1>
+----
+<1> (Optional) You can add these options if you need to gather more information from the OADP and the SR-IOV Operators.
+
+[id="ztp-image-based-upgrade-troubleshooting-manual-cleanup_{context}"]
+== AbortFailed or FinalizeFailed error
+
+Issue::
+During the finalize stage or when you stop the process at the `Prep` stage, normally, {lcao} cleans up the following resources:
+
+* Stateroot that is no longer required
+* Precaching resources
+* OADP CRs
+* `ImageBasedUpgrade` CR
+
+If the {lcao} fails to perform the above steps, it transitions to the `AbortFailed` or `FinalizeFailed` states.
+The condition message and log show which steps failed.
+
+.Example error message
+[source,yaml]
+----
+message: failed to delete all the backup CRs. Perform cleanup manually then add 'lca.openshift.io/manual-cleanup-done' annotation to ibu CR to transition back to Idle
+      observedGeneration: 5
+      reason: AbortFailed
+      status: "False"
+      type: Idle
+----
```

Comment:
Is this meant to be indented with the other text? You can probably accomplish that with an open block :thinking: